src/trader.js

```diff
@@ -945,7 +917,30 @@ async function UpdateOpenTrades() {
     })
 }
 
+async function UpdateMarginPairs() {
+    return new Promise((resolve, reject) => {
+        axios
+            .get(
+                "https://www.binance.com/gateway-api/v1/friendly/margin/symbols"
+            )
+            .then((res) => {
+                let btconly = res.data.data.filter(
+                    (obj) => obj.isMarginTrade == true && obj.quote == "BTC"
+                )
+                let list = btconly.map((obj) => obj.symbol)
+                margin_pairs = list.sort()
+                console.log("Margin Pairs:", margin_pairs)
+                resolve(true)
```

Comment:
Is there a reason `resolve` isn't called with the margin pairs array, but with just `true` instead, expect it being done like this in other parts of the code?
I'd propose to resolve with the actual data and reject with the error message below.